core/src/test/java/org/helsinki/paytrail/request/tokenization/PaytrailGetTokenRequestTest.java

```diff
@@ -0,0 +1,50 @@
+package org.helsinki.paytrail.request.tokenization;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.gson.Gson;
+import lombok.extern.slf4j.Slf4j;
+import org.helsinki.paytrail.PaytrailClient;
+import org.helsinki.paytrail.PaytrailCommonTest;
+import org.helsinki.paytrail.mapper.PaytrailGetTokenResponseMapper;
+import org.helsinki.paytrail.response.tokenization.PaytrailGetTokenResponse;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
+
+@Slf4j
+public class PaytrailGetTokenRequestTest extends PaytrailCommonTest {
+
+    PaytrailGetTokenResponseMapper mapper;
+
+    public PaytrailGetTokenRequestTest() {
+        this.mapper = new PaytrailGetTokenResponseMapper(new ObjectMapper());
+    }
+
+    @Test
+    void getToken() throws ExecutionException, InterruptedException {
+        PaytrailClient client = new PaytrailClient(merchantId, secretKey);
+
+        // Requires a valid checkout tokenization id
```

Comment:
I added some comments to describe how to use this test. I thought it would be useful to have here just in case. I tested the functionality thorugh this test with a valid checkout tokenization id.